nomad/grapl-core.nomad

```diff
@@ -0,0 +1,428 @@
+                                local_bind_port = 5080
+                            }
+
+                            # Connect this Alpha to Zero followers
+                            dynamic "upstreams" {
+                                iterator = zero_follower
+                                for_each = [for zero_follower in local.dgraph_zeros: zero_follower]
+
+                                content {
+                                    destination_name = "dgraph-zero-follower-${zero_follower.value.id}-grpc-private"
+                                    local_bind_port = zero_follower.value.grpc_private_port
+                                }
+                            }
+
+                            # Connect this Alpha to Other Alphas (but not to itself, obviously)
+                            dynamic "upstreams" {
+                                iterator = alpha_peer
+                                for_each = [for alpha_peer in local.dgraph_alphas: alpha_peer if alpha_peer.id != alpha.value.id]
+
+                                content {
+                                    destination_name = "dgraph-alpha-${alpha_peer.value.id}-grpc-private"
+                                    local_bind_port = alpha_peer.value.grpc_private_port
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+
+            service {
+                name = "dgraph-alpha-${alpha.value.id}-grpc-public"
+                port = "${alpha.value.grpc_public_port}"
+
+                connect {
+                    sidecar_service { }
+                }
+            }
+
+            service {
+                name = "dgraph-alpha-${alpha.value.id}-http"
+                port = "${alpha.value.http_port}"
+
+                connect {
+                    sidecar_service { }
+                }
+            }
+        }
+    }
+
+    group "grapl-graph-merger" {
+        count = var.graph_mergers
+
+        network {
+            mode = "bridge"
+        }
+
+        task "graph-merger" {
+            driver = "docker"
+
+            config {
+                image = "${var.container_registry}/graph-merger:${var.tag}"
+                args = ["/graph-merger"]
+            }
+
+            env {
+                REDIS_ENDPOINT = "${var.redis_endpoint}"
+                MG_ALPHAS = "${local.alpha_grpc_connect_str}"
+                GRAPL_SCHEMA_TABLE = "${var.schema_table_name}"
+                AWS_REGION = "${var.aws_region}"
+                # https://github.com/grapl-security/grapl/blob/18b229e824fae99fa2d600750dd3b17387611ef4/pulumi/grapl/__main__.py#L165
+                DEST_BUCKET_NAME = "subgraphs-merged-bucket"
+                SOURCE_QUEUE_URL = "${var.aws_sqs_url}/${var.aws_account_id}/graph-merger-queue"
+                DEAD_LETTER_QUEUE_URL = "${var.aws_sqs_url}/${var.aws_account_id}/graph-merger-dead-letter-queue"
```

Comment:
Same thing applies to these queue names.

(This naming issue applies to all the other containers, as well, so I won't repeat myself.)